api/src/apps/mod.rs

```diff
@@ -2,7 +2,7 @@
  * ========================LICENSE_START=================================
```

Comment:
Due to a hint of `cargo clippy` the file `api/src/apps/apps.rs` has been renamed to `api/src/apps/mod.rs`. Unfortunately, git does not recognize the diff.